utils/vision_utils.py

```diff
@@ -0,0 +1,95 @@
+import time
+import logging
+from datetime import datetime
+import os
+from typing import List
+
+logger = logging.getLogger(__name__)
+
+
+def validate(val_loader, model, criterion, args):
+    from trailmet.utils import accuracy, AverageMeter
+    import torch
+    logger = logging.getLogger(__name__)
+    name = "_".join(
+        [
+            "validate",
+            datetime.now().strftime("%b-%d_%H-%M-%S"),
+        ]
+    )
+    os.makedirs(f"{os.getcwd()}/logs/Validate", exist_ok=True)
+    logger_file = f"{os.getcwd()}/logs/Validate/{name}.log"
+    logging.basicConfig(
+        filename=logger_file,
+        format="%(asctime)s - %(levelname)s - %(name)s - %(message)s",
+        datefmt="%m/%d/%Y %H-%M-%S",
+        level=logging.INFO,
+    )
```

Comment:
we can use set logger if needed but I assume this is an intermediate step (within vision) and a logger is expected to have been initialised.

Reply:
wasnt using that file removed completely